From d60c8095d3bcb2fb09c4be5c0f083f5d80ce3dca Mon Sep 17 00:00:00 2001
From: Michal Simek <michal.simek@xilinx.com>
Date: Mon, 6 Jan 2014 16:50:28 +0100
Subject: [PATCH 077/182] tty: uartps: Fix kernel-doc errors in the driver

This patch comes from:
  https://github.com/Xilinx/linux-xlnx.git

Signed-off-by: Michal Simek <michal.simek@xilinx.com>
(cherry picked from commit dd63f77f96fe0e7f72963b04036309ec1675f05b)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/tty/serial/xilinx_uartps.c |    4 +++-
 1 files changed, 3 insertions(+), 1 deletions(-)

diff --git a/drivers/tty/serial/xilinx_uartps.c b/drivers/tty/serial/xilinx_uartps.c
index f619ad5..807cfb2 100644
--- a/drivers/tty/serial/xilinx_uartps.c
+++ b/drivers/tty/serial/xilinx_uartps.c
@@ -1040,6 +1040,8 @@ static struct uart_port xuartps_port[2];
  * xuartps_get_port - Configure the port from the platform device resource
  *			info
  *
+ * @id: Port id
+ *
  * Returns a pointer to a uart_port or NULL for failure
  **/
 static struct uart_port *xuartps_get_port(void)
@@ -1101,7 +1103,7 @@ static void xuartps_console_putchar(struct uart_port *port, int ch)
 
 /**
  * xuartps_console_write - perform write operation
- * @port: Handle to the uart port structure
+ * @co: Console handle
  * @s: Pointer to character array
  * @count: No of characters
  **/
-- 
1.7.5.4

